<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>1006</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,1" columnstretch="2,1,2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CL Mirror Remote Control</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="xy_groupBox">
     <property name="title">
      <string>XY</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="x_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>110</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
     <widget class="QPushButton" name="x_down_pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="x_up_pushButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>--&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="y_down_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>v</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="y_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>140</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
     <widget class="QPushButton" name="y_up_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QComboBox" name="xy_step_comboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="z_groupBox">
     <property name="title">
      <string>Z</string>
     </property>
     <widget class="QPushButton" name="z_up_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="z_down_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>v</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="z_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="z_step_comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="angle_groupBox">
     <property name="title">
      <string>Pitch and Yaw</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Yaw</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="pitch_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pitch_down_pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;-- Pitch</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pitch_up_pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Pitch --&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="yaw_down_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Yaw
v</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Pitch</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="yaw_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
     <widget class="QPushButton" name="yaw_up_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>^
Yaw</string>
      </property>
     </widget>
     <widget class="QComboBox" name="angle_step_comboBox">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SEM Stage</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="sem_xy_groupBox">
     <property name="title">
      <string>XY</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sem_x_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>110</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
     <widget class="QPushButton" name="sem_x_down_pushButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sem_x_up_pushButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>--&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sem_y_down_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>v</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sem_y_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>140</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
     <widget class="QPushButton" name="sem_y_up_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sem_xy_step_comboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sem_read_pos_pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>370</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Read SEM Stage Position</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="sem_z_groupBox">
     <property name="title">
      <string>Z</string>
     </property>
     <widget class="QPushButton" name="sem_z_up_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sem_z_down_pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>v</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sem_z_pos_doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sem_z_step_comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
